main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+func calculateSum(number int) int {
```

Comment:
This is not needed. Instead, create an endpoint which can accept an (id, SDP) pair (for now you can simulate SDP using any string), and return the SDP when it is queried.
eg.
Build two endpoints:
1. `POST /` which accepts a JSON object which contains the id and an SDP string.
2. `GET /<id>` which returns the SDP stored with `id` or 404 if not found.